third-party/chpl-venv/Makefile

```diff
@@ -5,15 +5,13 @@ endif
 CHPL_MAKE_HOST_TARGET = --host
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
-PYTHON = $(shell which python)
+# do not include ./Makefile.include from Makefile.base
```

Comment:
This comment needs to be expanded to say why Makefile.include isn't included in Makefile.base